package.json

```diff
@@ -29,11 +29,70 @@
       "no-shadow": [
         "error",
         {
-          "allow": ["resolve"]
+          "allow": [
+            "resolve"
+          ]
         }
       ],
       "no-use-before-define": "warn",
       "import/no-unresolved": "warn"
     }
+  },
+  "dependencies": {
+    "underscore": "^1.9.2"
+  },
+  "scripts": {
+    "start": "node script.js",
+    "build": "npm build",
+    "test": "npm start"
+  }
+}
+{
```

Comment:
The following looks like a copy from some starter app, this has no place here and breaks `package.json` as from this place onward it breaks the json format.